scraper/src/tests/default_strategy/get_anchor_test.py

```diff
@@ -107,3 +107,23 @@ def test_no_anchor(self):
 
         # Then
         assert actual is None
+
+    def test_linking_anchor(self):
+        # Given
+        strategy = get_strategy()
+        strategy.dom = lxml.html.fromstring("""
+        <html><body>
+            <h1>Foo</h1>
+            <h2 id='_docusaurus'>Bar</h2>
```

Comment:
If we want to reproduce the actual case, we should rather wrap this in a `div#__docusaurus` ([see a Docusaurus website source](https://v2.docusaurus.io/)).

Reply:
I added another test since the test also test the way we navigate through the DOM. It implies testing the strategy itself.